config.toml

```
@@ -1,10 +1,10 @@
-baseURL = 'https://hnolzen.github.io/swingconnection/'
+baseURL = 'https://swingconnection-leipzig.github.io/swingconnection/'
```

```suggestion
baseURL = 'https://swingconnection-leipzig.github.io/'
```
For staging we need no subpath.